Block/Adminhtml/Timeline/Index.php

```diff
@@ -55,36 +56,39 @@ class Index extends Template
      */
     private $config;
 
+    /**
+     * @var CronDataService
+     */
+    private $cronDataService;
```

Comment:
You can use strict type instead.

Reply:
Technically indeed, though this module should still be compatible with Magento 2.3. which still supports PHP 7.3

Reply:
You might use psalm to enforce type checks? (Would also solve issues like #3